lxd/instance_snapshot.go

```diff
@@ -379,15 +379,15 @@ func instanceSnapshotHandler(d *Daemon, r *http.Request) response.Response {
 
 	switch r.Method {
 	case "GET":
-		return snapshotGet(s, inst, snapshotName)
+		return snapshotGet(s, inst)
 	case "POST":
 		return snapshotPost(s, r, inst, containerName)
 	case "DELETE":
-		return snapshotDelete(s, r, inst, snapshotName)
+		return snapshotDelete(s, r, inst, containerName)
```

Comment:
Please can you add a separate commit to change the `containerName` var to `instName` - as we use "instance" to refer to both "container" and "virtual machine" now.

Reply:
Because we can get back to the parent instance name using:

```go
parent, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
```

I don't think we need to have the parent instance name passed in as a separate argument at all.
Instead each function can ascertain the parent name from the `snapInst` passed into it.

Reply:
Ah, went through a lot of the code, but didn't saw `api.GetParentAndSnapshotName(instName)`. Will use it instead then.

Reply:
Thanks for the inputs, resolved.